cmd/tendermint/commands/run_node.go

```diff
@@ -65,6 +65,16 @@ func AddNodeFlags(cmd *cobra.Command) {
 		"consensus.create_empty_blocks_interval",
 		string(config.Consensus.CreateEmptyBlocksInterval),
 		"The possible interval between empty blocks")
+
+	// db flags
+	cmd.Flags().String(
+		"db_backend",
+		config.DBBackend,
+		"Database backend: goleveldb | cleveldb | boltdb")
+	cmd.Flags().String(
+		"db_path",
```

Comment:
```suggestion
		"db_dir",
```

db_dir is used in config.toml

Reply:
Done